### ISSUE TITLE
[ter-indent] add new options for arrays and objects

### ISSUE THREAD

**Issue by jmlopez-rod** [buzinas/tslint-eslint-rules#160]
Since the last time I updated the rule there have been new changes, see:

https://github.com/eslint/eslint/compare/0643bfeff68979044ca57a2b392d855d18be7d08...dec3ec6a1d1ded6f931a0b9e31c324e3fea50550

I'll try to get these in with a couple of other rules for next minor release.

**Comment by blakeembrey** [comment]
@jmlopez-rod Is this related to the issues I'm seeing in `ter-indent` when using it like:

```ts
.update(
  {},
  {
    returnChanges: 'always'
  }
)
```

**Comment by jmlopez-rod** [comment]
@blakeembrey What's the configuration and errors you are using with the code? It could be, I'm gonna say the rule is complaining on `{ returnChanges: 'always' }` parameter?

**Comment by blakeembrey** [comment]
> I'm gonna say the rule is complaining on { returnChanges: 'always' } parameter?

Yes, sorry - it is. The rule is https://github.com/blakeembrey/tslint-config-standard/blob/6d0d2f5a968e36f7b763188d32f1b019cee85c52/tslint.js#L63-L69.

**Comment by mlippert** [comment]
This was the enhancement request I was about to file. I could use the Array and Object indent rule options that eslint has. I wanted to use this rule in tslint.yml:
```yaml
    ter-indent:
        severity: error
        options:
            - 4
            -
                SwitchCase: 1
                FunctionDeclaration:
                    parameters: first
                    body: 1
                FunctionExpression:
                    parameters: first
                    body: 1
                MemberExpression: off
                CallExpression:
                    arguments: first
                ArrayExpression: first
                ObjectExpression: first
                ImportDeclaration: first
```
so that this passes:
```js
    return {
               name,
               text: `You have to do ${name}`
           };

```

**Comment by dmnrmr** [comment]
Any updates?
